plugins/aws_sts/db/migrate/20181001083613_add_aws_sts_arn_to_stages.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddAwsStsArnToStages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :stages, :aws_sts_iam_role_arn, :string
+    add_column :stages, :aws_sts_iam_role_session_duration, :integer
```

Comment:
would prefer not having nil numbers, maybe `default: 900, null: false`

Reply:
if not, then make sure that saving `""` produces nil and not 0 ...

Reply:
Add just `default: 900, null: false` to the migration results in: `SQLite3::ConstraintException: NOT NULL constraint failed` when a nil value is submitted in the form - which is surprising, I expected the DB to apply the default value. So I've added a before save to set the default too.